62.CAD/main.cpp

```diff
@@ -4,19 +4,18 @@
 #include "../common/CommonAPI.h"
 #include "nbl/ext/FullScreenTriangle/FullScreenTriangle.h"
 
-static constexpr bool DebugMode = false;
-static constexpr bool FragmentShaderPixelInterlock = true;
-
 enum class ExampleMode
 {
 	CASE_0, // Simple Line, Camera Zoom In/Out
 	CASE_1,	// Overdraw Fragment Shader Stress Test
 	CASE_2, // NOT USED
 	CASE_3, // CURVES AND LINES
+	CASE_4, // ARC LENGTH
 };
 
-constexpr ExampleMode mode = ExampleMode::CASE_3;
-
+constexpr ExampleMode mode = ExampleMode::CASE_4;
+static constexpr bool DebugMode = false;
+static constexpr bool FragmentShaderPixelInterlock = (mode == ExampleMode::CASE_4) ? false : true;
```

Comment:
this is only for debugging though, you need to set this to previous `true` always

Reply:
done